third_party/silabs/SiWx917_sdk.gni

```diff
@@ -257,6 +257,7 @@ template("siwx917_sdk") {
       "SL_MATTER_GN_BUILD=1",
       "SILABS_LOG_OUT_UART=${sl_uart_log_output}",
       "SUPPORT_CPLUSPLUS=1",
+      "SLI_SI91X_LWIP_HOSTED_NETWORK_STACK=1",
```

Comment:
still this is not optional to LWIP, can we do that?

Reply:
This is mandatory now, will move it and other files as well during socket bringup